### PR TITLE
Fix/package visible

### PR DESCRIPTION
**Pull request by adhiraj23zelthy**

### CODE REVIEW

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10470524660). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10666655268). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723633365061.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723633365061.min.js:1]: @adhiraj23zelthy we can remove the build here, it will get pushed automatically once PR is merged

File: backend/src/zango/assets/app_panel/js/build.1723633365061.min.js.LICENSE.txt
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723633365061.min.js.LICENSE.txt:1]: @adhiraj23zelthy  we can remove this file

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

File: backend/src/zango/assets/app_panel/js/build.1723633365061.min.js
Comment (adhiraj23zelthy) [reply backend/src/zango/assets/app_panel/js/build.1723633365061.min.js:1]: done

File: backend/src/zango/assets/app_panel/js/build.1723633365061.min.js.LICENSE.txt
Comment (adhiraj23zelthy) [reply backend/src/zango/assets/app_panel/js/build.1723633365061.min.js.LICENSE.txt:1]: done

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10667400389). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: frontend/src/pages/appPackagesManagement/components/AppTable/columns.jsx [hunk 1]
```diff
@@ -73,28 +74,47 @@ function columns({ dispatch }) {
...
+					className={`w-fit min-w-max rounded-[15px] text-center font-lato text-[14px] font-normal capitalize leading-[20px] tracking-[0.2px] text-[#5048ED]`}
+				>
+					View Details
+				</span>
+				<DetailEyeIcon />
+			</button>
+		</div>
+	
+}
+
```
Comment (sukuna28) [inline frontend/src/pages/appPackagesManagement/components/AppTable/columns.jsx:120]: @adhiraj23zelthy let's create separate component for this, name it `ViewDetailButton.jsx`

**Review by sukuna28 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10667954293). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: frontend/src/pages/appPackagesManagement/components/AppTable/columns.jsx (hunk 1 above)
Comment (adhiraj23zelthy) [reply frontend/src/pages/appPackagesManagement/components/AppTable/columns.jsx]: done

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10681436214). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10683117394). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10700514808). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723136204750.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723136204750.min.js:1]: @adhiraj23zelthy Please revert this change

File: backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js:1]: @adhiraj23zelthy Please revert this change

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10718756554). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723136204750.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723136204750.min.js:1]: @adhiraj23zelthy we can keep the file, you can copy it from the main branch and add it

File: backend/src/zango/assets/app_panel/js/.gitignore
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/.gitignore:1]: please remove this file
@adhiraj23zelthy

File: backend/src/zango/assets/app_panel/js/build.1723136204750.min.js
Comment (adhiraj23zelthy) [reply backend/src/zango/assets/app_panel/js/build.1723136204750.min.js:1]: done

File: backend/src/zango/assets/app_panel/js/.gitignore
Comment (adhiraj23zelthy) [reply backend/src/zango/assets/app_panel/js/.gitignore:1]: done

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10732370177). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10733739979). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Review by shahharsh176 — APPROVED** [review]

**Review by sukuna28 — APPROVED** [review]
